packages/battery_monitor/package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>battery_monitor</name>
+  <version>1.0.0</version>
+  <description>Reads the status from the new Duckietown batteries</description>
```

Comment:
do they have a name and version number ? Would be nice to put that instead of "new", which will be wrong in a year. @tanij ?